utilities/write_batch_with_index/write_batch_with_index.cc

```diff
@@ -452,7 +459,10 @@ bool WriteBatchWithIndex::Rep::UpdateExistingEntryWithCfId(
   }
   WriteBatchIndexEntry* non_const_entry =
       const_cast<WriteBatchIndexEntry*>(iter.GetRawEntry());
-  obsolete_offsets.push_back(non_const_entry->offset);
+  if (UNLIKELY(last_sub_batch_offset <= non_const_entry->offset)) {
```

Comment:
why this is unlikely? I think if we get here we find a duplicated key and it will be likely to be a new duplicated key. Do I miss something?

Reply:
You are right. Let me change it to LIKELY.